### PR TITLE
Test

### PR DESCRIPTION
**Pull request by KevinDang12**
Test

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ const cors = require('cors');
 app.use(express.json());
 app.use(cors());
 
+// Test Comment
```
Comment (brandang) [inline server/server.js]: Please remove this comment

File: server/server.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ const cors = require('cors');
 app.use(express.json());
```
Comment (brandang) [inline server/server.js]: Good solution I like JSON

**Review by brandang — CHANGES_REQUESTED** [review]
LGTM, just a few comments

**Comment by KevinDang12** [comment]
Removed the Test Comment

**Comment by brandang** [comment]
> Removed the Test Comment

Cool lgtm

**Review by brandang — APPROVED** [review]
cool lets merge

**Pull request merged**